demo/src/Components/Buttons/DropdownToggle.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import T from 'prop-types'
+import classNames from 'classnames'
+
+const DropdownToggle = ({ handleToggle, children, className }) => (
```

Comment:
I'd do this so that just the dropdown portion of the button opens the dropdown, rather than the entire button.
So if you click on the box with the dropdown then the dropdown appears, if you click on the remaining area of the button then the standard behaviour is exhibited - hence it's segmented with two behaviours.

Reply:
done